app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
```

Comment:
No trailing `if/else`
Use multi-line if instead to emphasize code branches

Reply:
Thanks Elle, need to talk with you about this one as Rubocop wanted it this way and gave me a violation for needing a guard clause instead of the if/else.